src/component/Header/CréerMonCocktail.jsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+function CréerMonCocktail() {
+  return (
+    <h1 className="links">Hello from Créer mon Cocktail</h1>
```

Comment:
doesn't look like a link... maybe you should rename your class into "color-white" to make it reusable not only for links ;)